common/supplemental/supplementalData.xml

```diff
@@ -4535,30 +4535,35 @@ XXX Code for transations where no currency is involved
         <calendar type="islamic">
           <calendarSystem type="lunar" />
           <eras>
+              <era type="-1" end="622-7-14" code="islamic-inverse" aliases="bh"/>
               <era type="0" start="622-7-15" code="islamic" aliases="ah"/>
           </eras>
         </calendar>
         <calendar type="islamic-civil">
           <calendarSystem type="lunar" />
           <eras>
+              <era type="-1" end="622-7-15" code="islamic-civil-inverse" aliases="bh"/>
```

Comment:
I gave it the type -1 because type 0 is already used for the AH era. Please let me know if I should instead:

1. Use a positive number like 1 (and leave AH as 0)
2. Change BH to 0, AH to 1, and update all the XML files that currently have translations for AH at type 0 to instead be at type 1

Reply:
@sffc Usually this number is used as an array index in ICU. I guess I would be in favor of your option #2 above. We would need to add a migration note for ICU (and for CLDR), and probably update some ICU code. I think we allow years to be negative, but I do not think we allow eras to be.